src/style-auro-calendar-cell.scss

```diff
@@ -1,73 +1,175 @@
+  font-size: var(--ds-text-body-size-default, $ds-text-body-size-default);
 
   user-select: none;
 
   &.disabled {
-    color: var(--auro-color-text-secondary-on-dark);
+    color: var(--ds-color-text-secondary-inverse, $ds-color-text-secondary-inverse);
 
     pointer-events: none;
   }
 
   &.selected {
-    border-color: var(--auro-color-background-darker);
+    border-color: var(--ds-color-background-darker, $ds-color-background-darker);
 
-    background-color: var(--auro-color-background-darker);
+    background-color: var(--ds-color-background-darker, $ds-color-background-darker);
     box-shadow: 0 .1px 5px rgba(0, 0, 0, .12), 0 5px 10px rgba(0, 0, 0, .1);
-    color: var(--auro-color-background-lightest);
+    color: var(--ds-color-background-lightest, $ds-color-background-lightest);
+
+    &.rangeDepartDate::before {
+      @include auro_breakpoint($max: $ds-grid-breakpoint-sm) {
+        @include in-range-pseudo-elem;
+
+        width: calc(calc(100vw / 7) / 2);
+
+        transform: translate(0%, -50%);
+      }
+    }
+
+    &.rangeReturnDate::before {
+      @include auro_breakpoint($max: $ds-grid-breakpoint-sm) {
+        @include in-range-pseudo-elem;
+
+        width: calc(calc(100vw / 7) / 2);
+
+        transform: translate(-100%, -50%);
+      }
+    }
   }
 
   &.inRange {
     &:before {
-      position: absolute;
-      z-index: -1;
-      top: 50%;
-      left: 50%;
+      @include in-range-pseudo-elem;
+
+      @include auro_grid-breakpoint--sm {
+        width: var(--ds-size-600, $ds-size-600);
+      }
+    }
+  }
+}
 
-      display: block;
-      width: var(--auro-size-600);
-      height: var(--auro-size-600);
+.dateSlot {
+  display: flex;
+  flex-direction: column;
 
-      background-color: var(--auro-color-background-lighter);
+  font-size: var(--ds-text-body-size-xxs, $ds-text-body-size-xxs);
+}
 
-      content: '';
-      transform: translate(-50%, -50%);
-    }
+::slotted([slot^="date_"]) {
+  color: var(--ds-color-text-link-default, $ds-color-text-link-default);
+
+  position: absolute;
+  top: 80%;
+  left: 50%;
+
+  width: 80%;
+
+  overflow: hidden;
+  white-space: nowrap;
+
+  transform: translate(-50%, -50%);
+}
+
+:host([disabled]) ::slotted([slot^="date_"]) {
+  color: var(--ds-color-text-disabled-default, $ds-color-text-disabled-default);
+}
+
+::slotted(auro-icon) {
+  max-height: 24px;
+  max-width: 24px;
+}
+
+:host([renderForDateSlot]) {
+  .buttonWrapper {
+    position: relative;
+    width: 100%;
+  
+    top: 5px;
+  }
+  
+  .currentDayMarker {
+    position: relative;
+    padding-bottom: 5px;
+    top: -8px;
+  }
+}
+
+@include auro_grid-breakpoint--sm {
+  .day {
+    width: var(--ds-size-600, $ds-size-600);
+    height: var(--ds-size-800, $ds-size-800);
+
+    font-size: var(--ds-text-body-size-lg, $ds-text-body-size-lg);
+  }
+
+  .dateSlot {
+    font-size: var(--ds-text-body-size-xs, $ds-text-body-size-xs);
   }
 }
 
 @media (hover: hover) {
```

Comment:
Something is going on with the hover affect with range support - demoed live to @jordanjones243 earlier.

Reply:
Fixed by adding a function in the `classMap` of `auro-calendar-cell` called `isInRange`.